src/main/kotlin/com/example/ajaxproject/dto/request/ChatDTO.kt

```diff
@@ -1,6 +1,12 @@
 package com.example.ajaxproject.dto.request
 
-data class ChatDTO(
+
+data class GroupChatDTO(
     val senderId: String,
+    val message: String,
+    override val chatId: String
+) : ChatDTO
+
+interface ChatDTO {
```

Comment:
What about name
```suggestion
interface Identifiable {
```
?

Reply:
Looked really bad and should be fixed)